tests/test_cases/test_array/test_array.py

```diff
@@ -363,6 +363,8 @@ async def test_discover_all(dut):
         pass_total = 571
     elif cocotb.SIM_NAME.lower().startswith("ghdl"):
         pass_total = 56
+    elif cocotb.LANGUAGE in ["vhdl"] and cocotb.SIM_NAME.lower().startswith("modelsim"):
```

Comment:
How do we differentiate between FLI and VHPI here?

Reply:
I see we use `VHDL_GPI_INTERFACE` below.

Reply:
You're pointing in the right direction. After some more edits I forgot to run against the fli interface as well; I'll update this PR to avoid breaking those tests.